stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/attachments/content/ImageAttachmentContent.kt

```diff
@@ -60,6 +62,7 @@ import io.getstream.chat.android.uiutils.extension.hasLink
  * and the onLongItemClick handler.
  */
 @OptIn(ExperimentalFoundationApi::class)
+@Suppress("MagicNumber", "LongMethod")
```

Comment:
Instead of suppressing, we should generate the baseline to include this.

I would also just pull out the aspect ratio "magic numbers" to constants, as it will be a bit more descriptive